### ISSUE TITLE
Chrome does not open on Ubuntu 14.04 (seems to be not replicable)

### ISSUE THREAD

**Issue by balwant-sd** [kelaberetiv/TagUI#303]
When i use some automation flow work of web, like visit or login a portal and download some reports. 

It all execute in backend. I want when these tasks performs it will open the chrome then and every step.

Please Help me @Aussiroth

**Comment by ryzalk** [comment]
Hey @balwant-sd. Have you tried running with the 'chrome' option, as stated [here](https://github.com/kelaberetiv/tagui#command-line)?

For example:
`$ tagui samples/1_yahoo chrome`

**Comment by balwant-sd** [comment]
yes.

it will give error

`./tagui: line 313: curl: command not found`

**Comment by ryzalk** [comment]
Perhaps 'curl' is not installed. Try running the following line:
`sudo apt-get install curl`

**Comment by balwant-sd** [comment]
Not it sits idle.

![45](https://user-images.githubusercontent.com/44574296/47846047-af6d5b00-ddec-11e8-9261-0ae8ee259365.png)

**Comment by balwant-sd** [comment]
Issue occurring while using firefox
```
pericent@pericent-desktop:~/tagui/src$ ./tagui samples/1_yahoo firefox
1541069900896	addons.xpi	WARN	List of valid built-in add-ons could not be parsed.: [Exception... "Component returned failure code: 0x80520012 (NS_ERROR_FILE_NOT_FOUND) [nsIXPCComponents_Utils.readUTF8URI]"  nsresult: "0x80520012 (NS_ERROR_FILE_NOT_FOUND)"  location: "JS frame :: resource://gre/modules/addons/XPIProvider.jsm :: startup :: line 2111"  data: no] Stack trace: startup()@resource://gre/modules/addons/XPIProvider.jsm:2111
callProvider()@resource://gre/modules/AddonManager.jsm:206
_startProvider()@resource://gre/modules/AddonManager.jsm:654
startup()@resource://gre/modules/AddonManager.jsm:807
startup()@resource://gre/modules/AddonManager.jsm:2802
observe()@jar:file:///usr/lib/firefox/omni.ja!/components/addonManager.js:66
1541069900918	addons.xpi	WARN	No list of valid system add-ons found.
Error during the script execution
[Exception] TypeError: ioService.newChannel is not a function  filename:resource://slimerjs/slUtils.jsm  line:180

Stack trace:
0:readChromeFile@resource://slimerjs/slUtils.jsm:180:19
launchMainScript@resource://slimerjs/slLauncher.jsm:109:23
startup@chrome://slimerjs/content/slimerjs.js:20:9
onload@chrome://slimerjs/content/slimerjs.xul:1:1

```

Now what i do ??

**Comment by Aussiroth** [comment]
Hi, if your firefox is on version 60 or later, TagUI will not work with firefox option. The framework TagUI uses for automating firefox, slimerjs, has not yet been patched to be able to work properly with firefox version 60 and later.
Regarding using chrome, can you check if the src/tagui_chrome.log has any text inside, or is it empty?

**Comment by balwant-sd** [comment]
Hey @Aussiroth 

src/tagui_chrome.log is empty.

**Comment by kensoh** [comment]
Hi Alvin and Ryzal, I made a PR on patch-289-295 branch. It resolves PHP 7 warning messages at issue https://github.com/kelaberetiv/TagUI/issues/295#issuecomment-435471557 and also the dynamic wait issue for #289.

For the missing PhantomJS at issue #295 and the Chrome issue #303 here, it may be a non-issue for now as it can't be replicated. I borrowed an Ubuntu 14.04 and ran the yahoo sample using chrome option smoothly (using the Linux zip). Until more Ubuntu users report similar findings so that attempts to replicate can be made, there is probably nothing that can be done to troubleshoot.
